pull_request_template.md

```diff
@@ -1,7 +1,8 @@
-Mantis: <Enter link to Mantis>
-Problem / Feature: See Mantis
-Solution: See Mantis
-Testing: See Mantis
+Jira: <Enter link to Jira>
```

Comment:
If we format as per: https://github.com/integrations/jira#link-issues-in-github github will auto insert the link I believe.

Reply:
Test jira link:
[ABIP-601]

Reply:
Works in fb4_subsytems